i18nextBrowserLanguageDetector.js

```diff
@@ -162,15 +162,15 @@
     }
   };
 
-  var url = {
-    name: 'url',
+  var folder = {
```

Comment:
wouldn't make path or route more sense -> there is no real folder (no filesystem)

Reply:
Done. 👍